### PR TITLE
Remove heartbeat connector from Mirror Maker 2 examples

### PR DESCRIPTION
**Pull request by scholzj**
### Type of change

- Task

### Description

This PR removes the `heartbeat` connector from out Mirror Maker 2 examples. The heartbeat connector in MM2 does not work really well in our examples:
* It connects directly to the target cluster, so it is not really mirrored and does not add much value
* The purpose does not seem clear to the users. They do not really understand that it is optional or how to use it. They often think that this is some mandatory part without which MM2 will not work.
* It has some value in active-active mirroring when it is mirrored. But the user experience seems weird since you configure it in the _wrong_ `KafkaMirrorMaker2` resource

This PR impacts only the examples. It is still part of the code and users can enable it. It also remains to be documented. We should aim to improve the documentation and better explain when it might be useful. But that is out of scope for this PR.

### Checklist

- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally

### CODE REVIEW

File: packaging/examples/security/scram-sha-512-auth/mirror-maker-2.yaml [hunk 1]
```diff
@@ -202,7 +202,7 @@ spec:
           name: "*"
         operations:
           - Read
-      # MirrorHeartbeatConnector
+      # MirrorHeartbeatConnector => needed only if the connector is enabled
```
Comment (mimaison) [inline packaging/examples/security/scram-sha-512-auth/mirror-maker-2.yaml:205]: Should we remove that too?
Same below

Comment (scholzj) [reply packaging/examples/security/scram-sha-512-auth/mirror-maker-2.yaml:205]: I would not remove it -> I think it is useful to track this. But we can for example consider commenting out the ACLs.

**Review by mimaison — APPROVED** [review]

**Pull request merged**